files/jupyter-library/basic-time-series-plotting-template.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import seaborn as sns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Metadata saved to output plots\n",
+    "METADATA = {\n",
+    "    \"Contributor\": \"Alexander Gude\",\n",
+    "    \"Rights\": \"CC-BY-SA 4.0\"\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Show plots inline\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Helper functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We should be able to just set rcParams, expect Jupyter has a bug:\n",
+    "# https://github.com/jupyter/notebook/issues/3385\n",
+    "#\n",
+    "# So we have to call this function every time we want to plot.\n",
+    "\n",
+    "def setup_plot(title=None, xlabel=None, ylabel=None):\n",
+    "    \"\"\"Set up a simple, single pane plot with custom configuration.\n",
```

Comment:
This is horrible. You really have to do a list of lines?

Reply:
I python notebooks are stored as JSON... So yes?